docs/integrations/aws.md

```diff
@@ -435,6 +435,23 @@ For the above example, the objects in S3 will be saved with tags: `my_key1=my_va
 
 For more details on tag restrictions, please refer [User-Defined Tag Restrictions](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html).
 
+### S3 AccessPoints
```

Comment:
nit: prefer to not use variable name in code but full english words, like "S3 access points"

Reply:
changed it "S3 Access Points", made **_Access Points_**  to match https://github.com/apache/iceberg/blob/master/docs/integrations/aws.md#s3-access-control-list